scripts/timescaledb/before-create.sql

```diff
@@ -0,0 +1,24 @@
+-- Check if one of the timescaledb schemas are present in the database, while the extension is missing
+-- Note: this list assumes any new schemas added to the extension should be present here as well.
+DO $$
+BEGIN
+    IF EXISTS(
+    SELECT
+        1
+    FROM
+        pg_catalog.pg_namespace
+            WHERE nspname IN (
+                '_timescaledb_catalog',
```

Comment:
or should we "take over" _timescaledb_% instead of hard-coding schemas in this list?